torch/serialization.py

```diff
@@ -330,6 +330,11 @@ def _validate_privateuse1_device(location, backend_name):
                            'please use torch.load with map_location=torch.device(\'cpu\') '
                            'to map your storages to the CPU.')
     device_module = getattr(torch, backend_name)
+    if hasattr(device_module, '_utils'):
```

Comment:
Please make this a fully documented function on the device module so that users know how to implement this.

Reply:
add documented for the method， please check